libunwind/src/Registers.hpp

```diff
@@ -29,7 +29,9 @@ struct __attribute__((aligned(16))) fake_capability {
   char bytes[16];
 };
 typedef struct fake_capability fake_capability_t;
+#define _LIBUNWIND_UNDEF_CAPABILITY ((fake_capability_t){{0}})
```

Comment:
This is only used in C++ code, can we just use `{{0}}`?

Reply:
Yes, that should work. Pushed. Thanks!